docker/launch.sh

```diff
@@ -197,8 +219,10 @@ fi
 
 if [ ${NVIDIA} = false ]; then
     DOCKER="docker"
-else
+elif [[ ${NVIDIA} = true && ${NVIDIA_VERSION} = 1 ]]; then
     DOCKER="nvidia-docker"
```

Comment:
if ${NVIDIA_VERSION} = 2, then it is "docker"?

Reply:
Yes